### DISCUSSION TITLE
Location name in DAGSTER_CONTAINER_CONTEXT executed ECS Task

### DISCUSSION THREAD

**Discussion by tomsej** [dagster-io/dagster#20339] [Q&A] [answered]
Hi, I am using **ECS Run Launcher** with multiple locations. In every launched container I would like to have a name of location that contains executed code. I have tried to specify it with `DAGSTER_CONTAINER_CONTEXT` in the location:
```            
"ecs": {
    "container_name": "launcher-task",
     "env_vars": ["LOCATION=xxx"],
 }
```
But it does not seem to work. I would be grateful for any advice.

**Comment by garethbrickman** [comment]
Possibly #12621 answers this?

**Reply by tomsej** [reply]
Thanks @garethbrickman , that’s what I was trying to do. I set the “env_vars” into the locations ENVs, but they didn’t get passed into the running ECS task. But e.g. `task_definition_arn` works as expected.

**Reply by gibsondan** [reply]
Hi @tomsej - your configuration looks correct to me. How are you checking whether the env vars are set the way you expect? I would expect them to be visible on the task definition - is it possible to share the task definition JSON for your run task from the ECS console?

**Reply by gibsondan** [reply, marked as answer]
Specifically in the task definition JSON i'd be curious what you see under containerDefinitions[0]["environment"] (the list of environment variables for the "dagster" container).

Is the "container_name" key working the way you expect?

**Reply by tomsej** [reply]
Ohh, I feel really stupid. Think I had a typo in `container_name`. Now it works like a charm. Thank you!
